package.json

```diff
@@ -29,16 +29,9 @@
   },
   "homepage": "https://github.com/silverstripe/silverstripe-cms#readme",
   "devDependencies": {
-    "@silverstripe/webpack-config": "^0.2.5",
-    "babel-core": "^6.7.4",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-preset-react": "^6.5.0",
-    "eslint": "^2.7.0",
-    "eslint-config-airbnb": "^6.2.0",
-    "eslint-plugin-react": "^4.3.0"
+    "@silverstripe/webpack-config": "file:../../webpack-config"
```

Comment:
For development purposes, to replace with the proper `0.3.0` version when that is published on the `webpack-config` package

Reply:
Can you change it back to "^0.3.0" since changes on `webpack-config` has been merged?